library/core/api/src/main/kotlin/de/lise/fluxflow/api/workflow/continuation/WorkflowContinuation.kt

```diff
@@ -34,6 +39,12 @@ data class WorkflowContinuation<TWorkflowModel, out TContinuation>(
         )
     }
 
+    fun withReplacementScope(replacementScope: Set<WorkflowElement>): WorkflowContinuation<TWorkflowModel, TContinuation> {
```

Comment:
Can we provide an additional overload accepting the scope as a vararg, so that developers don't need to always do the awkward Set construction.

e.g.
```kotlin
fun withReplacementScope(vararg replacementScope: WorkflowElement): WorkflowContinuation<TWorkflowModel, TContinuation> {
    return withReplacementScope(setOf(*replacementScope))
}
```